.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
 script:
 - npm run lint
 - npm run test
+- npm run codecov
```

Comment:
is codecov enought? To run tests and codecoverage?

Reply:
yep, `npm run test` builds the coverage report and codecov uploads it. See https://github.com/Rintel/Breminale-Web/pull/7/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15